### PR TITLE
[wip] Increasing netpol timeout to 3s

### PR DESCRIPTION
**Pull request by knabben**
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by juampynr** [comment]
@knabben why is this change needed? Are you running into race conditions? If so, where?

File: test/e2e/network/netpol/model.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func NewModel(namespaces []string, podNames []string, ports []int32, protocols [
 
 // GetProbeTimeoutSeconds returns a timeout for how long the probe should work before failing a check, and takes windows heuristics into account, where requests can take longer sometimes.
 func (m *Model) GetProbeTimeoutSeconds() int {
-	timeoutSeconds := 1
+	timeoutSeconds := 3
```
Comment (mattfenwick) [inline test/e2e/network/netpol/model.go:93]: Echoing https://github.com/kubernetes/kubernetes/pull/104087#issuecomment-891728927 -- what prompted this?

And also, if we're going to use the same value across OSs, we can drop the windows conditional, and drop the `GetProbeTimeoutSeconds` method altogether in favor of a named constant.

Comment (knabben) [reply test/e2e/network/netpol/model.go:93]: just testing the CI, this PR wont be merged

**Pull request closed without merging**